src/LiLiQ-Rplus-1.1.xml

```diff
@@ -0,0 +1,258 @@
+                 commerce du concédant, autre que celle requise
+                 permettant d'identifier la provenance du
+                 logiciel.
+          </p>
+        </li>
+        <li>
+          <b>7.</b>
+          <p>Garanties
+            <br/>Sauf mention contraire, le concédant distribue le
+                 logiciel sans aucune garantie, aux risques et
+                 périls de l'acquéreur de la copie du logiciel,
+                 et ce, sans assurer que le logiciel puisse
+                 répondre à un besoin particulier ou puisse
+                 donner un résultat quelconque.
+          </p>
+          <p>Sans lier le concédant d'une quelconque manière, rien
+             n'empêche un licencié d'offrir ou d'exclure des
+             garanties ou du support.</p>
+        </li>
+        <li>
+          <b>8.</b>
+          <p>Responsabilité
+            <br/>Le licencié est responsable de tout préjudice
+                 résultant de l'exercice des droits accordés
+                 par la licence.
+          </p>
+          <p>Le concédant ne saurait être tenu responsable du
+             préjudice subi par le licencié ou par des tiers, pour
+             quelque cause que ce soit en lien avec la licence et
+             les droits qui y sont accordés.</p>
+        </li>
+        <li>
+          <b>9.</b>
+          <p>Résiliation
+            <br/>La présente licence est résiliée de plein droit dès
+                 que les droits qui y sont accordés ne sont pas
+                 exercés conformément aux termes qui y sont
+                 stipulés.
+          </p>
+          <p>Toutefois, si le défaut est corrigé dans un délai de 30
+             jours de sa prise de connaissance par la personne en
+             défaut, et qu'il s'agit du premier défaut, la licence
+             est accordée de nouveau.</p>
+          <p>Pour tout défaut subséquent, le consentement exprès du
+             concédant est nécessaire afin que la licence soit
+             accordée de nouveau.</p>
+        </li>
+        <li>
+          <b>10.</b>
+          <p>Version de la licence
+            <br/>Le Centre de services partagés du Québec, ses
+                 ayants cause ou toute personne qu'il désigne,
+                 peuvent diffuser des versions révisées ou
+                 modifiées de cette licence. Chaque version
+                 recevra un numéro unique. Si un logiciel est
+                 déjà soumis aux termes d'une version
+                 spécifique, c'est seulement cette version qui
+                 liera les parties à la licence.
+          </p>
+          <p>Le concédant peut aussi choisir de concéder la licence
+             sous la version actuelle ou toute version ultérieure,
+             auquel cas le licencié peut choisir sous quelle
+             version la licence lui est accordée.</p>
+        </li>
+        <li>
+          <b>11.</b>
+          <p>Divers</p>
+        </li>
+      </list>
```

Comment:
hard to say from looking at the original license. can you understand enough to make a call?  if it's not clear, it's probably fine to put it after the second paragraph